lux-lib/src/operations/sync.rs

```diff
@@ -267,6 +268,8 @@ async fn do_sync(
         project_lockfile.sync(dest_lockfile.local_pkg_lock(), lock_type);
     }
 
+    luarc::update_luarc(args.config);
+
```

Comment:
todo: We should probably add some snapshot tests.

If you prefer, I can do this as a follow-up after merging - let me know :smile: